recipes/zserio/all/conanfile.py

```diff
@@ -0,0 +1,96 @@
+import os
+import shutil
+
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import collect_libs, copy, download, get
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+
+required_conan_version = ">=1.54.0"
+
+class ZserioConanFile(ConanFile):
+    name = "zserio"
+    description = "Zserio C++ Runtime Library"
+    license = "BSD-3-Clause"
+    url = "https://github.com/conan-io/conan-center-index/"
+    homepage = "https://zserio.org"
+    topics = ("zserio", "cpp", "c++", "serialization")
+    package_type = "static-library"
```

Comment:
```suggestion
    package_type = "library"
```

This recipe packages more than a static library, so it should be classified as `library`. It's the default, but can include more than only libraries. To understand more about traits, please, read https://docs.conan.io/2/reference/conanfile/methods/requirements.html#package-type-trait-inferring

Reply:
Done. But I'm raising ConanInvalidConfiguration for `shared` option.